plugin/src/main/java/org/opensearch/ml/jobs/processors/MLStatsJobProcessor.java

```diff
@@ -151,4 +179,67 @@ public void onFailure(Exception e) {
             }
         });
     }
+
+    private void collectAgentMetrics() {
+        // check if `.plugins-ml-agent` index exists
+        if (!clusterService.state().metadata().indices().containsKey(ML_AGENT_INDEX)) {
+            log.info("Skipping ML agent metrics collection - ML agent index not found");
+            return;
+        }
+
+        SearchRequest searchRequest = new SearchRequest(ML_AGENT_INDEX);
+        SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
+        searchSourceBuilder.size(10_000);
```

Comment:
can you find if there is an existing constant somewhere to reuse? otherwise using a new constant to be reusable for this common parameter.